full-backend-tests/src/test/java/org/graylog/plugins/views/SearchSyncIT.java

```diff
@@ -149,6 +150,40 @@ void testRequestStoredSearchWithGlobalOverrideKeepingOnlySingleSearchType() thro
                 .body("results.f1446410-a082-4871-b3bf-d69aa42d0c97.search_types", hasKey("01c76680-377b-4930-86e2-a55fdb867b58"));
     }
 
+    @ContainerMatrixTest
+    void testThatQueryOrderStaysConsistentInV1() {
+        given()
+                .spec(requestSpec)
```

Comment:
`given()
                .config(sut.withGraylogBackendFailureConfig())
                .spec(requestSpec)`
and
`                .then()
                .log().ifStatusCodeMatches(not(201))
                .statusCode(201)
`

would create better logging in case of underlying errors on Jenkins

Reply:
:white_check_mark: